CODEOWNERS

```diff
@@ -12,4 +12,7 @@ integration-tests/* @ed255
 
 # Prs done to `zkevm-circuits/state-circuit` will require an extra review from @miha-stopar
 zkevm-circuits/src/state_circuit/ @miha-stopar
-zkevm-circuits/src/state_circuit.rs @miha-stopar
\ No newline at end of file
+zkevm-circuits/src/state_circuit.rs @miha-stopar
+
+# Prs done in `testool` will requiere an extra review from @adria0
+testool/ @adria0
```

Comment:
I think we have to change the line 1 from "@appliedzkp/zkevm-reviewers" to "@privacy-scaling-explorations/zkevm-reviewers"